usr/share/rear/conf/default.conf

```diff
@@ -334,6 +334,28 @@ CDROM_SIZE=20
 # does not automatically recreate the rescue/recovery system.
 CHECK_CONFIG_FILES=( '/etc/drbd/' '/etc/drbd.conf' '/etc/lvm/lvm.conf' '/etc/multipath.conf' '/etc/rear/' '/etc/udev/udev.conf' )
 
+# FILES_TO_PATCH_PATTERNS is a space-separated list of shell glob patterns.
+# Files that match are eligible for a final migration of UUIDs and other
+# identifiers after recovery (if the layout recreation process has led
+# to a change of an UUID or a device name and a corresponding change needs
+# to be performed on restored configuration files ).
+# See finalize/GNU/Linux/280_migrate_uuid_tags.sh
+# The funny [] around the first letter make sure that shopt -s nullglob removes this file from the list if it does not exist
+# the files without a [] are mandatory, like fstab FIXME: but below there is [e]tc/fstab not etc/fstab - why?
+
```

Comment:
I was thinking several times about the "files without a [] are mandatory" part.
Now I think this text is plain wrong because we skip such "mandatory" files
when they do not exist in current master code e.g.
in finalize/GNU/Linux/250_migrate_disk_devices_layout.sh
and in finalize/GNU/Linux/280_migrate_uuid_tags.sh
```
for restored_file in [b]oot/{grub.conf,menu.lst,device.map} ...
do
    # Silently skip directories and file not found:
    test -f "$restored_file" || continue
```
I think it is right to skip non existent files there because
those scripts are not meant to test if restored files are missing
even if files that are normally mandatory like etc/fstab (or etc/crypttab)
are missing in the restored target system.

Perhaps there are systems without fstab?
If yes `[e]tc/fstab` is right.
I know there is systemd.mount but I do not know
if it is possible to run a system without fstab.
Regardless whether or not it is possible to run a system without fstab
it is currently not ReaR's job to verify whether or not mandatory files
have been restored form the backup.
ReaR considers things are OK when the backup restore program call
exited with zero exit code or at least when there is no real error exit code
for the latter see e.g. restore/TSM/default/400_restore_with_tsm.sh

So I suggest to remove the whole line
```
# the files without a [] are mandatory, like fstab FIXME: but below there is [e]tc/fstab not etc/fstab - why?
```

Reply:
I understood it as "strange" not as "amusing", i.e. https://en.wiktionary.org/wiki/funny#Adjective meaning 2. and not 1. I can replace "funny" with "strange" or "weird" if you think that it looks better.

Reply:
I think it is neither "strange" nor "amusing" nor "weird" because
it is the right method to get what is intended in this case.

Reply:
Addressed in 52b28f7b50eb99fbb933e04c5e3be75afeacc894